pkg/cmd/codespace/create.go

```diff
@@ -112,11 +113,16 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		return errors.New("there are no available machine types for this repository")
 	}
 
+	vscsTarget := os.Getenv("VSCS_TARGET")
+	vscsTargetUrl := os.Getenv("VSCS_TARGET_URL")
```

Comment:
@mislav no they are not standard. They are mostly used only by the codespaces team under certain conditions when we want to point to a different platform environment, this is to improve our development innerloop. In the linked issue I recommended this approach to avoid polluting flags for optional codespaces-team specific configuration.

Reply:
These aren't ever meant to be passed in by normal users, only for internal development.  I'm not sure if Cobra supports hidden command-line flags - that would be another option.